admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneUserServiceImpl.java

```diff
@@ -127,7 +127,7 @@ public ZoneUserExtnDto createZoneUserMapping(ZoneUserDto zoneUserDto) {
 
 			// Throws exception if not found
 			zoneservice.getZone(zoneUserDto.getZoneCode(), languageUtils.getDefaultLanguage());
-
+			zu.setUserId(zu.getUserId().toLowerCase());

```

Comment:
When lowercase conversion is taken care of in the queries. why are we converting to lowercase here?

Reply:
Moved this change to dto level as suggested.